src/foam/nanos/cron/Schedulable.js

```diff
@@ -0,0 +1,72 @@
+/**
+ * @license
+ * Copyright 2021 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+ foam.CLASS({
+  package: 'foam.nanos.cron',
+  name: 'Schedulable',
+  extends: 'foam.nanos.cron.Cron',
+
+  mixins: [
+    'foam.nanos.auth.CreatedAwareMixin',
+    'foam.nanos.auth.CreatedByAwareMixin'
+  ],
+
+  implements: [
+    'foam.core.ContextAgent',
+    'foam.nanos.auth.ServiceProviderAware'
+  ],
+
+  properties: [
+    {
+      name: 'daoKey',
+      value: 'schedulableDAO'
+    },
+    {
+      class: 'String',
+      name: 'eventDaoKey',
+      value: 'schedulableEventDAO'
+    },
+    {
+      name: 'schedule',
+      of: 'foam.nanos.cron.Schedule',
+      view: {
+        class: 'foam.u2.view.FObjectView',
+        of: 'foam.nanos.cron.SimpleIntervalSchedule'
+      }
+    },
+    {
+      class: 'String',
+      name: 'objectDaoKey'
+    },
+    {
+      class: 'FObjectProperty',
+      of: 'FObject',
+      name: 'objectToSchedule'
+    },
+    {
+      class: 'Reference',
+      of: 'foam.nanos.auth.ServiceProvider',
+      name: 'spid',
```

Comment:
Is this property needed? Won't it be inherited from the ServiceProviderAware interface?

Reply:
Removed